### ISSUE TITLE
Is it possible to add metaContext?

### ISSUE THREAD

**Issue by THPubs** [timbrandin/autoform-slingshot#3]
One way to allow the meta context (https://github.com/CulturalMe/meteor-slingshot#add-meta-context-to-your-uploads) is to let the user define a new slingshot upload `uploader = new Slingshot.Upload(directiveName)` and not do it in the plugin. Any idea?

**Comment by timbrandin** [comment]
@THPubs you mean that the user defined an uploader in the schema rather than the directive?

**Comment by timbrandin** [comment]
I guess we could modify this code to support something like that:
https://github.com/timbrandin/autoform-slingshot/blob/master/lib/client/autoform-slingshot.coffee#L119-L129

**Comment by THPubs** [comment]
Yep! that should do it! We can pass a custom variable through `afFieldInput` for example `customDirective = true`. Then in the plugin if this variable is undefined or true, we won't initiate the upload?

Or

We can pass `metaContext` through `afFieldInput` right?

I tried to do the first one but didn't got time to complete. Need to study the code a bit :-)
